### PR TITLE
added new third party host - btcpaynet.com

### PR DESCRIPTION
**Pull request by iOSDeveloperSK**

### CODE REVIEW

File: ThirdPartyHosting.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ In layman words, think of this feature as a payment processor factory which allo
 
 ## The list of BTCPay third-party hosts
 
+* [btcpaynet.com](https://btcpaynet.com)
```
Comment (pavlenex) [inline ThirdPartyHosting.md:11]: The list is in alphabetical order. So your entry needs to be after btcpayjungle.

Comment (iOSDeveloperSK) [reply ThirdPartyHosting.md]: Done. Thank you :-)
